CHANGELOG.md

```diff
@@ -9,10 +9,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ### Changed
 
 - **CUMULUS-2312** - RDS Migration Epic Phase 3
+  - **CUMULUS-2801**
+    - Move `getFilesExistingAtLocation`from api granule, update granules put
```

Comment:
Nit: It would be more clear if note specified the function move from api models granule to api lib.

Reply:
An excellent suggestion, thank you: 

133b543e7a